digdag-standards/src/test/resources/io/digdag/standards/operator/py/output.json

```diff
@@ -0,0 +1 @@
+{"subtask_config": {}, "export_params": {}, "store_params": {}, "error": {"class": "NameError", "message": "name 'printaa' is not defined", "backtrace": ["Traceback (most recent call last):\n", "  File \"./runner.py\", line 168, in <module>\n    result = callable_type(**args)\n", "  File \"/Users/you.yamagata/cs_work/CS-5411-py-tokenize/test1.py\", line 4, in func1\n    printaa(\"func1 called\")\n", "NameError: name 'printaa' is not defined\n"]}}
```

Comment:
Also this file includes your local file path info, how about updating a more general one here?

Reply:
> Better to add a new line at the end of this file?
This files is generated by runner.py and it does not contain new line. So keeping it may be better.

> This file includes your local file path info, how about updating a more general one here?
Oh, thanks. I will mask the information.

Reply:
I fixed it.